ui/shared/Revision.jsx

```diff
@@ -62,15 +93,16 @@ export class Revision extends React.PureComponent {
     const {
       revision: { comments, author, revision },
       repo,
+      bugSummaryMap,
     } = this.props;
     const comment = comments.split('\n')[0];
-    const { clipboardVisible } = this.state;
+    const bugMatches = comment.match(/-- ([0-9]+)|bug.([0-9]+)/gi);
+    const { clipboardVisible, ready, tooltipOpen } = this.state;
     const { name, email } = parseAuthor(author);
     const commitRevision = revision;
     const commentColor = this.isBackout(comment)
       ? 'text-danger'
       : 'text-secondary';
-
```

Comment:
We prefer to keep a blank line between `const` declarations and implementation code. So let's leave this.